pulsar-client-admin/src/main/java/com/yahoo/pulsar/client/admin/Brokers.java

```diff
@@ -67,4 +67,13 @@
      * @throws PulsarAdminException
      */
     Map<String, NamespaceOwnershipStatus> getOwnedNamespaces(String cluster, String brokerUrl) throws PulsarAdminException;
+    
+    /**
+     * Update value of any dynamic {@link ServiceConfiguration} 
+     * 
+     * @param key
+     * @param value
+     * @throws PulsarAdminException
+     */
+    void updateConfiguration(String configName, String configValue) throws PulsarAdminException;
```

Comment:
Just renaming. I'm not 100% sure about the name, just wanted to make clear that it doesn't update the config in a broker, rather it writes config in ZK that supersedes whatever all the brokers have locally.